specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/examples/AgentPoolsCreate_Flex.json

```diff
@@ -0,0 +1,153 @@
+{
+  "parameters": {
+    "api-version": "2023-08-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "agentPoolName": "agentpool1",
+    "parameters": {
+      "properties": {
+        "orchestratorVersion": "1.9.6",
+        "osType": "Linux",
+        "tags": {
+          "name1": "val1"
+        },
+        "nodeLabels": {
+          "key1": "val1"
+        },
+        "nodeTaints": [
+          "Key1=Value1:NoSchedule"
+        ],
+        "type": "Flex",
+        "flexProfile": {
+          "scaleProfile": {
+            "manual": {
+              "machines": [
+                {
+                  "sizes": "Dsv3, Dv3",
+                  "count": 5
+                },
+                {
```

Comment:
I thought we agree to have only one item for now?

Reply:
Yes I put it in the api proposal. This is to illustrate we keep this as a list and will do the enforcement via validation at the beginning on len(machines) == 1.
We might still want to support some cases that people need exact 5 VM1 and 3 VM2.

Reply:
i think the fact that this is an array is self explanatory. imo, having this example would make it look like legitimate.